### DISCUSSION TITLE
Doesn't work via universal control

### DISCUSSION THREAD

**Discussion by Sollembum78** [noah-nuebling/mac-mouse-fix#344] [Questions] [answered]
I have iMac and Macbook both on my table, and I use my a4tech "bloody" mouse wired to imac. With universal control I can click on both computers with this wired mouse. If I use scroll reverser on both - it will work the same, both mac that has mouse connected, and the one with universal control. But with this app (which is superior), reverse smooth scrolling doesn't seem to work.

**Comment by noah-nuebling** [comment]
Hi @Sollembum78, thanks for the feedback!

That's really interesting that Scroll Reverser still works. 
I don't have an iPad to test on so I won't be able to work on this soon, but I would also love to see this!

**Comment by noah-nuebling** [comment, marked as answer]
Marking this as answered to clean things up. I've moved the discussion to #277 so it's all concentrated in one place. Let me know if you have more questions or ideas!
